src/components/Pages/Submit/Form/Toolbar.tsx/IconButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC, ReactComponentElement, ReactElement } from "react";
+
+import { ChainedCommands, Editor } from "@tiptap/react";
+
+type commands =
+  | "blur"
+  | "setParagraph"
+  | "toggleBlockquote"
+  | "toggleBold"
+  | "toggleBulletList"
+  | "toggleCode"
+  | "toggleOrderedList"
+  | "toggleStrike"
+  | "toggleItalic"
+  | "toggleUnderline"
+  | "unsetAllMarks"
+  | "undo";
+const IconButton: FC<{
```

Comment:
I mostly did this class to make it dry, It's unclear if it's an actual improvement.